src/nvim/testdir/test_options.vim

```diff
@@ -353,3 +353,24 @@ func Test_shortmess_F()
   set shortmess&
   bwipe
 endfunc
+
+func Test_shortmess_F2()
+  e file1
+  e file2
+  call assert_match('file1', execute('bn', ''))
```

Comment:
This fails for some reason, even though doing the steps manually works.

    TEST_FILE=test_options.res make oldtest

Reply:
Does this require `set shortmess-=F` for nvim?

Reply:
`set shortmess&` from the test above includes "F" with Neovim.
The `set shortmess-=F` from setup.vim is only done initially/once.

Reply:
still fails with this though?!

```diff
diff --git i/src/nvim/testdir/test_options.vim w/src/nvim/testdir/test_options.vim
index 155d67ce9..b8e83877e 100644
--- i/src/nvim/testdir/test_options.vim
+++ w/src/nvim/testdir/test_options.vim
@@ -357,6 +357,7 @@ endfunc
 func Test_shortmess_F2()
   e file1
   e file2
+  set shortmess-=F
   call assert_match('file1', execute('bn', ''))
   call assert_match('file2', execute('bn', ''))
   set shortmess+=F
```

Reply:
Any `:edit` or `msg_silent` patches we're missing?